erpnext/stock/doctype/stock_entry/stock_entry.py

```
@@ -123,6 +123,14 @@ class StockEntry(StockController, SubcontractingInwardController):
 		pick_list: DF.Link | None
 		posting_date: DF.Date | None
 		posting_time: DF.Time | None
+		print_barcode_color: DF.Color | None
+		print_font_for_barcodes: DF.Literal["Helvetica, Arial, sans-serif", "Arial, sans-serif", "Verdana, sans-serif", "Tahoma, sans-serif", "\"Times New Roman\", Times, serif", "\"Roboto\", sans-serif", "\"Lato\", sans-serif", "Consolas, Helvetica, Arial, sans-serif"]
```

Ruff (0.14.5)

Undefined name `Helvetica`

(F821)